dotcom-rendering/fixtures/manual/liveBlock.ts

```diff
@@ -0,0 +1,24 @@
+export const baseBlock: Block = {
```

Comment:
```suggestion
export const liveBlock: Block = {
```

It's always nice if the name of the file matches the name of the export. It makes searching and refactoring easier later on.

Reply:
Absolutely, renamed!